addons/l10n_in/views/res_partner_views.xml

```diff
@@ -8,12 +8,15 @@
         <field name="arch" type="xml">
             <xpath expr="//field[@name='vat']" position="attributes">
                 <attribute name="string">GSTIN</attribute>
```

Comment:
I know it is not in this PR, but it is something that has bothered us: it should not change the label of the VAT number, but it should just add a vat_label on the country.  If you could remove this line...

Reply:
Done